docs/reference/extensions/django-framework.rst

```diff
@@ -43,6 +54,43 @@ application. In the following example we use it to specify ``libpq-dev``:
         # list required packages or slices for your Django application below.
         - libpq-dev
 
+.. _django-async-deps:
+
+``parts`` > ``django-framework/async-dependencies``
+===================================================
+
+In order to be able to use async Gunicorn workers you need to use
+``django-framework/async-dependencies`` part instead of
+``django-framework/dependencies`` part.
+
+To use this just uncomment the following lines:
```

Comment:
This line implies to me that the ``rockcraft.yaml`` file automatically contains this snippet as part of the initialization. Is that true?

In any event, just in case the snippet is removed in the future, I would rephrase this line to say something like 
```
In the ``rockcraft.yaml``, add the following lines:
```